package.json

```diff
@@ -62,6 +61,7 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
+    "ember-decorators": "^6.1.1",
```

Comment:
looks like it's not necessary here, because of migration to glimmer (ember-cli-babel handle decorators just fine, and ember-decorators related to decorators usage inside classic ember components)

Reply:
Thanks :)